simapp/v2/app.go

```diff
@@ -220,5 +222,18 @@ func (app *SimApp[T]) Close() error {
 }
 
 func ProvideRootStoreConfig(config runtime.GlobalConfig) (*root.Config, error) {
-	return serverstore.UnmarshalConfig(config)
+	cfg, err := serverstore.UnmarshalConfig(config)
+	if err != nil {
+		return nil, err
+	}
+	iavlV2Opts := iavlv2.DefaultTreeOptions()
+	iavlV2Opts.MinimumKeepVersions = int64(cfg.Options.SCPruningOption.KeepRecent)
+	iavlV2Opts.EvictionDepth = 22
+	iavlV2Opts.HeightFilter = 1
+	iavlV2Opts.CheckpointInterval = 60
+	iavlV2Opts.PruneRatio = 0.8
+	iavlv2.SetGlobalPruneLimit(1)
+	cfg.Options.IavlV2Config = iavlV2Opts
```

Comment:
should we set this by default and allow people to override it? feels like extra boilerplate that teams will mainly copy.

Reply:
Yea I'll move these defaults to store/v2